cassandra-mesos-scheduler/src/test/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/SeedManagerTest.java

```diff
@@ -0,0 +1,91 @@
+package io.mesosphere.mesos.frameworks.cassandra.scheduler;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Joiner;
+import org.apache.mesos.state.InMemoryState;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static io.mesosphere.mesos.frameworks.cassandra.CassandraFrameworkProtos.ExternalDc;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class SeedManagerTest {
+    private SeedManager seedManager;
+    private String jsonResponse = seedsJson(Collections.<String>emptyList());
+
+    @Before
+    public void before() {
+        InMemoryState state = new InMemoryState();
+
+        PersistedCassandraFrameworkConfiguration config = new PersistedCassandraFrameworkConfiguration(
+                state,
+                "name",
+                60,
+                30,
+                "2.1",
+                0.5,
+                1024,
+                1024,
+                512,
+                1,
+                1,
+                "role",
+                "",
+                false,
+                true,
+                "RACK1",
+                "DC1",
+                Arrays.asList(ExternalDc.newBuilder()
+                        .setName("dc")
+                        .setUrl("http://dc")
+                        .setSeedFetchTime(-SeedManager.FETCH_PERIOD)
+                        .build())
+        );
+
+        seedManager = new SeedManager(config) {
```

Comment:
Can the original code be refactors so that this inline overriding doesn't have to take place?  This override shadows a method that is actually pretty important.